### DISCUSSION TITLE
Using with reanimated 2.2.0

### DISCUSSION THREAD

**Discussion by Psyborgs-git** [software-mansion/react-native-screens#966] [Q&A] [answered]
Can I use Animated.createAnimatedComponent() on <Screens\> to animate props like activityState and styles ?

**Comment by WoLewicki** [comment, marked as answer]
I think the thing you are asking for is connected to this PR: https://github.com/software-mansion/react-native-screens/pull/890. It makes it possible to wrap `Screen` with `createAnimatedComponent` from `react-native-reanimated` and use the transition progress of the screen in your components. Am I right?

**Reply by Psyborgs-git** [reply]
yes kinda.
